geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraFeatureStore.scala

```diff
@@ -65,6 +66,12 @@ class CassandraFeatureStore(entry: ContentEntry) extends ContentFeatureStore(ent
   }
 
 
+  override def getInfo: ResourceInfo = {
```

Comment:
seems reasonable to me. definitely in geomesa-utils, I think that's the only module imported into all projects.